packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md

```diff
@@ -9,7 +9,7 @@ Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
 - Fix overlay notifications so they cover the area that their background blocks - [#5975](https://github.com/flutter/devtools/pull/5975)
 
 ## Inspector updates
-TODO: Remove this section if there are not any general updates.
+- Fix bug related to upcoming Flutter Multi View functionality - [#5986](https://github.com/flutter/devtools/pull/5986)
```

Comment:
I would argue that we don't need a release note for a bug that doesn't exist yet 🤔

Reply:
I was split on this, thanks for your input! :D